src/constants/types/Link.ts

```diff
@@ -1,4 +1,4 @@
-export interface Link {
-    title:string,
-    to:string,
-}
+export interface LinkElements {
```

Comment:
Please change naming convention. ([2,5,7] => [number,number,number] not [numbers, numbers, numbers]